### ISSUE TITLE
No visual indication of clause template search bar highlighting

### ISSUE THREAD

**Issue by Michael-Grover** [accordproject/cicero-ui#269]
If I type text in the clause template search bar and then attempt to highlight the text, there is no visual indication that the text is highlighted, however I know the text is highlighted because if I press backspace, highlighted text will be deleted. I just can't see what's highlighted.

![image](https://user-images.githubusercontent.com/40569793/74977876-90d40e80-53f9-11ea-9b1a-2742185e5d7e.png)

https://www.loom.com/share/75b31df8d96043d5911b844fdf2bb1ca

**Comment by adrijshikhar** [comment]
i would like to work on this issue

**Comment by Michael-Grover** [comment]
Hi @adrijshikhar , I've assigned you to the issue, thank you! Let me or @irmerk know if you have any questions. I will add more detail to the description of this issue

**Comment by Michael-Grover** [comment]
@adrijshikhar I added more detail and a video to the issue description

**Comment by adrijshikhar** [comment]
I have set up the repo `cicero-ui` but it does not look like what it is in the video. Am I missing something?

**Comment by Michael-Grover** [comment]
I'm not sure, @irmerk or @DianaLease may be able to help with this on Monday.

Until then I recommend checking out this information @jeromesimeon posted in the Accord Project Slack if you are interested in participating in Google Summer of Code with the Accord Project

For everyone who just joined: Welcome! Here are several useful pointers:
Google Summer of Code “How it Works” (including the timeline) https://summerofcode.withgoogle.com/how-it-works/
Google Summer of Code “Student Guide” https://google.github.io/gsocguides/student/
Accord Project organization page in GSoC https://summerofcode.withgoogle.com/organizations/5605109054046208/
Accord Project documentation https://docs.accordproject.org
Project ideas list https://github.com/accordproject/techdocs/wiki/Google-Summer-of-Code
We are looking forward to your project proposals! Feel free to suggest ideas for projects that are not on the list.

**Comment by adrijshikhar** [comment]
Thanks.

**Comment by ankingcodes** [comment]
@adrijshikhar I faced the same problem when I had set up this repo.
> Edit - template-studio-v2 uses cicero-ui for its components. So in order to fix assigned issue, i have to set up template-studio-v2 also, right?

Could you help me set this up ?

**Comment by irmerk** [comment]
@adrijshikhar @ankingcodes the README should likely be updated to better explain this, but the demo that runs locally by default in the repository is only the `ContractEditor`, not the `TemplateLibrary` component. 

`template-studio-v2` is the project which uses all these components at the moment. So local development of components *other than* `ContractEditor` will either take `template-studio-v2` or tinkering with the demo code.

**Comment by ankingcodes** [comment]
> tinkering with the demo code

@irmerk this turned out to be a safer route for me. 

> development of components other than ContractEditor will either take template-studio-v2

i did clone the `template-studio-v2` repo and set it up , but the README there also was unclear about how `npm link` works. I'd be glad if either the README is updated or if you could explain how that works here.

**Comment by adrijshikhar** [comment]
> @adrijshikhar @ankingcodes the README should likely be updated to better explain this, but the demo that runs locally by default in the repository is only the `ContractEditor`, not the `TemplateLibrary` component.
> 
> `template-studio-v2` is the project which uses all these components at the moment. So local development of components _other than_ `ContractEditor` will either take `template-studio-v2` or tinkering with the demo code.

I linked the development version of `cicero-ui` with `template-studio-v2` and made the changes. I am not able to get your point, can you please elaborate it a bit? Thanks
